lib/cloudwatchlogger/client.rb

```diff
@@ -10,7 +9,7 @@ def self.new(credentials, log_group_name, log_stream_name = nil, opts = {})
         raise LogGroupNameRequired, 'log_group_name is required'
       end
 
-      CloudWatchLogger::Client::AWS_SDK.new(credentials, log_group_name, log_stream_name, opts)
+      CloudWatchLogger::Client::AwsSDK.new(credentials, log_group_name, log_stream_name, opts)
```

Comment:
not sure about this class name. for its ruby gem, AWS uses `Aws::` as the namespace, but I'm not 100% comfortable with `SDK` here